import_token.go

```diff
@@ -133,6 +134,52 @@ func encryptNonce(key, value, iv string) (string, string, error) {
 	return base64.StdEncoding.EncodeToString(cipherText), iv, nil
 }
 
+// EncryptNonceWithCBC encrypts the nonce using the user's key-material
+// with CBC encrypter, it requires padding, here pkcs7 padding is used.
+// This is needed for Hyper Protect Crypto Services, since it supports only
+// CBC Encryption.
+func EncryptNonceWithCBC(key, value, iv string) (string, string, error) {
+	keyMat, err := base64.StdEncoding.DecodeString(key)
+	if err != nil {
+		panic(err)
```

Comment:
Pretty sure you want to `return nil, nil, fmt.Errorf("...")` like `encryptNonce()` above

Reply:
Good catch, thanks